edit_here_to_add_blocklists.json

```diff
@@ -147,7 +147,7 @@
     "loc": "custom_filters/normal/id12.txt",
     "name": "someonewhocares",
     "category": "Ads, Privacy",
-    "source": "https://someonewhocares.org/hosts/hosts",
+    "source": "https://someonewhocares.org/hosts/ipv6zero/hosts/",
```

Comment:
No need of changing this, server by default blocks both A and AAAA records for the domain present in the list.

Reply:
> 

Ok il switch both lists right away